sos/report/plugins/vectordev.py

```diff
@@ -0,0 +1,24 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin
+
+
+class VectorDev(Plugin):
```

Comment:
I think you must use `IndependentPlugin` instead of the base class `Plugin`, to allow the plugin on any distro.

Reply:
yep, fixed now.